README.md

```diff
@@ -13,6 +13,15 @@ of vulnerability reports.
 This project is developed by [Trail of Bits](https://www.trailofbits.com/) with
 support from Google. This is not an official Google product.
 
+## Features
+
+* Support for auditing both local environments and requirements-style files
+* Support for multiple vulnerability services (PyPI, OSV)
+* Support for emitting SBOMs as CycloneDX XML or JSON
+* Human and machine-readable output formats (columnar, JSON)
+* "Strict" and "dry-run" modes for high assurance and evaluation
```

Comment:
```suggestion
```
Not sure we're totally ready to use the words 'high assurance' in the context of this project 😉

Reply:
Fair!